xarray/core/variable.py

```diff
@@ -51,6 +53,7 @@
 NON_NUMPY_SUPPORTED_ARRAY_TYPES = (
     indexing.ExplicitlyIndexed,
     pd.Index,
+    pd.api.extensions.ExtensionArray,
```

Comment:
This should not be needed. We're working on deleting this variable. As long as `ExtensionDuckArray` passes the `is_duck_array` check, it should work.

Reply:
This is needed to get the variable into an `ExtensionDuckArray`.  What is the plan for deleting this variable?  I can try to follow it but for now the `ExtensionArray` needs to be wrapped internally just like `pd.Index` from what I can tell by `_maybe_wrap_data`.

Reply:
yeah I guess we need this for the pandas types :/